src/backend/src/services/risks.services.ts

```diff
@@ -0,0 +1,130 @@
+import { Role, User } from '@prisma/client';
```

Comment:
Yea I think this service is perfect. The only question I have is whether we want to make it a class we export, so we can have a 1:1 naming scheme for the controller to service methods (at least some of them)